docs/_posts/murat-gunay/2021-12-07-bert_token_classifier_chinese_ner_zh.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: model
+title: Chinese NER Model
+author: John Snow Labs
+name: bert_token_classifier_chinese_ner
+date: 2021-12-07
+tags: [chinese, token_classifier, bert, zh, open_source]
+task: Named Entity Recognition
+language: zh
+edition: Spark NLP 3.3.2
+spark_version: 2.4
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+- This model is imported from `Hugging Face`.
```

Comment:
- This model was imported from `Hugging Face` and it's been fine-tuned for traditional Chinese language, leveraging `Bert` embeddings and `BertForTokenClassification` for NER purposes.